src/csharp/Microsoft.Spark.Worker/Microsoft.Spark.Worker.csproj

```diff
@@ -5,9 +5,13 @@
     <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1</TargetFrameworks>
     <RootNamespace>Microsoft.Spark.Worker</RootNamespace>
-    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.Spark.Worker.UnitTest" />
```

Comment:
Nice, I didn't know this was possible. Would've saved me a lot of pain when I did the signing work.